packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,81 @@
+# Amount <!-- omit in toc -->
```

Comment:
The API decisions is already a part of #993, we can remove from this PR.

Reply:
+1

Reply:
@Priyanshu-C lets not keep this here. Chain this PR to Amount decisions so that this PR only contains the implementation.